README.rst

```diff
@@ -57,23 +59,108 @@ Travis CI and AppVeyor are both continuous integration services configured to be
 
 Make sure to enabled both services to work with your repository.
 
-Travis CI will be also used to deploy the application to `PyPI`_, for more details check the section `Release` on the README file from your project.
-In order to set up this configuration, it's necessary to encrypt you PyPi password on `.travis.yml`
+4.1 Activating Travis:
+
+1. Open https://travis-ci.org/ sign in with github account
+2. Access https://travis-ci.org/ESSS/<package_name>
+3. Click on Activate repository
+
+4.2 Activating Appveyor:
+
+1. Open https://www.appveyor.com/ and sign in with github account
+2. On the dashboard click on `New Project`
+3. Wait (no kidding, takes a time to load all projects).
+4. Search for the <project_name> and click on `Add`
+
+5. Setup deploy to PyPI
+
+Make sure to have an account on `PyPI`_ 
+
+Travis CI will be also used to deploy the application to `PyPI`_, for more details check the section ``Release`` on the README file from your project.
+In order to set up this configuration, it's necessary to encrypt you `PyPI`_ password on ``.travis.yml``
+
+Run the commands below on the root directory of your project.
 
 .. code-block:: bash
 
       1) travis encrypt --add deploy.password
       2) Type the password
-      3) Press Enter
-      4) On the new line Press CTRL+Z
+      
+      On Linux:
+         3) Press CTRL+D
+      Windows:
+         3) Press Enter
+         4) On the new line Press CTRL+Z
+
+The command line from ``travis`` will create a new entry called "Deploy" on your ``.travis.yml``  file. 
+
+.. code-block:: python
+
+      deploy:
+        provider: pypi
+        distributions: sdist bdist_wheel
+        user: <pypi user>
+        password:
+          secure: REPLACE
+        on:
+          tags: true
+          repo: ESSS/<package name>
+          python: 3.6
+  branches:
+    only:
+    - master
+    - "/v(\\d+\\.)*\\d/"
+  deploy:
+    password:
+      secure: <encrypted password>
+
+
+Copy the ``<encrypted password>`` and replace the ``REPLACE`` text with the encrypted password.
+
+Do not forget to delete the deploy and password newly created tags
+
+.. code-block:: python
+
+      deploy:
+        provider: pypi
+        distributions: sdist bdist_wheel
+        user: <pypi user>
+        password:
+          secure: <encrypted password>
+        on:
+          tags: true
+          repo: ESSS/<package name>
+          python: 3.6
+  branches:
+    only:
+    - master
+    - "/v(\\d+\\.)*\\d/"
+
+
+6. Set Up ReadTheDocs
+
+`ReadTheDocs`_ hosts documentation for the open source community.
 
+1. Log into your account at `ReadTheDocs`_ . If you don't have one, create one and log into it.
```

Comment:
Leave just _Log in to your ReadTheDocs account_ on this line. The rest is pretty obvious